api-helpers/contentCreatorFunctions.ts

```diff
@@ -0,0 +1,152 @@
+import { resolve } from 'path';
+import { URL } from 'url';
+
+import Boom from '@hapi/boom';
+import cheerio from 'cheerio';
+import Slugify from 'slugify';
+
+import { closeConnection, openConnection } from './db';
+
+const NEVER = new Date(0);
+
+type BlogData = {
+  readonly name: string;
+  readonly href: string;
+  readonly favicon: string;
+  readonly feeds: readonly Feed[];
+};
+
+type Feed = {
+  readonly type: string;
+  readonly url: string;
+  readonly title: string;
+};
+
+export const addContentCreator = async (url: string, email?: string) => {
+  const prisma = await openConnection();
+  try {
+    const { name, href, favicon, feeds } = await getBlogData(url);
+    return prisma.blog.create({
```

Comment:
Brakuje await

Reply:
Tutaj tak jakby zwracamy promise, więc jeśli coś użyje tej funkcji to tak czy siak musi dać await przed tą funkcją.

Reply:
Ale try catch nie złapie tego wyjątku, więc po co w ogóle try catch tutaj?

Reply:
Tak, masz rację.